src/client/types.ts

```diff
@@ -12,9 +12,12 @@ type Data = {
   output: {}
 }
 
+type AfterHook = (res: Response) => Promise<Response | void> | Response | void
```

Comment:
Similar to Ky's inferface, `AfterResponse` or `AfterResponseHook` would be more descriptive. Just a personal preference though.

Reply:
Descriptive is important. Let's make it `AfterResponseHook`.